lib/cldr/thor.rb

```diff
@@ -5,10 +5,16 @@ module Cldr
   class Thor < ::Thor
     namespace 'cldr'
 
-    desc "download [--source=http://unicode.org/Public/cldr/26/core.zip] [--target=./vendor]",
-         "Download and extract CLDR data from source to target dir"
+    desc "download [--version=34] [--target=./vendor] [--source=http://unicode.org/Public/cldr/34/core.zip]",
+        <<~DESCRIPTION
+          Download and extract CLDR data:
+            * Use the --version parameter to set the release version of the CLDR data to use
+            * Use the --target parameter to specify where on the filesystem to extract the downloaded data
+            * Use the --source parameter to override the location of the CLDR zip to download. Overrides --version
+        DESCRIPTION
     method_options %w(source -s) => :string,
-                   %w(target -t) => :string
+                   %w(target -t) => :string,
+                   %w(version -v) => :string
```

Comment:
Also need to add `options['version']` on line 21, right?

Reply:
Whoops, @trishrempel I looked at the wrong line. Yes, good spot